packages/vue-query/src/index.ts

```diff
@@ -0,0 +1,26 @@
+/* istanbul ignore file */
+
+export * from '@tanstack/query-core'
+
+export { useQueryClient } from './useQueryClient'
+export { VueQueryPlugin } from './vueQueryPlugin'
+
+export { QueryClient } from './queryClient'
+export { QueryCache } from './queryCache'
+export { MutationCache } from './mutationCache'
```

Comment:
do we need to re-export the core if you have your own wrappers? I mean the `query-core` also has a `QueryClient`, so what happens if we export this twice?

Reply:
The latter will overwrite the former. It works well.